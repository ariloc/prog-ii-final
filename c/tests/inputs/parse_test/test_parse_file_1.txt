   este   es el comienzo de una oracion, ,un poco  desprolija  
,y extrana. sin embargo, deberia ser un buen 1!er ejemplo
,para poder PROBAR LA
CAP
,A
!!CI
   DADDDDDD   
 de obtener REsuLtados CONsisTenTEESS.
 ---- FIN -----
.
  . termino todo.
todo.
todo
