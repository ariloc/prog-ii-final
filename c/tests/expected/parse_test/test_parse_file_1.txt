este es el comienzo de una oracion un poco desprolija y extrana
sin embargo deberia ser un buen er ejemplo para poder probar la cap a ci dadddddd de obtener resultados consistenteess
fin

termino todo
todo
todo
